.github/workflows/conformance.yml

```diff
@@ -108,7 +107,7 @@ jobs:
       uses: actions/checkout@v2
       with:
         repository: libjxl/conformance
-        ref: ce5a804a3ff834097b93c39d2f93e5445f6bf340
```

Comment:
This is actually the correct hash :) `master` on `libjxl/conformance` is not the most recent/up to date conformance test.

Fixing that...